index.html

```diff
@@ -0,0 +1,244 @@
+
+
+
+                </div>
+            </div>
+        </section>
+    </main>
+
+
+    <section id="about">
+        <div class="lets-grow">
+            <h3 class="lets-grow-title">Main Program</h3>
+            <ul class="programs">
+                <li class="Lecture">
+                    <img src="./img/Tools.png" alt="Tools">
+                    <h4>Lecture</h4>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+
+                </li>
+                <li class="cc-Exhibition">
+                    <img src="./img/Bulb.png" alt="Bulb">
+                    <h4>CC Exhibtion</h4>
+                    <p>Lorem ipsum dolor sit amet.</p>
+
+                </li>
+                <li class="forum">
+                    <img src="./img/Handshake.png" alt="Forum">
+                    <h4>Forum</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+
+                </li>
+                <li class="workshop">
+                    <img src="./img/Medal.png" alt="Medal">
+                    <h4>Workshop</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+
+                </li>
+                <li class="ignite">
+                    <img src="./img/Handshake.png" alt="Handshake">
+                    <h4> CC Ignite</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, corporis!</p>
+
+                </li>
+
+
+            </ul>
+            <div class="see-program"> <a href="#">See our Full Program</a>
+            </div>
+
+
+
+
+        </div>
+
+    </section>
+
+    <section class="ourspeaker" id="ourspeakers">
+        <h2 class="speaker-h2">Our SPEAKERS</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec malesuada tristique justo quis
+            ultrices. Morbi gravida dignissim lectus, et semper nulla varius a</p>
+        <div class="speakers" id="loadSpeakers">
+            <div class="speaker1" id="">
+                <div class="all-tiles">
+                    <!-- <img class="tiles" src="./img/tiles.png" alt="Tiles"> -->
+                    <img src="./img/speaker-1.png" alt="Mike Wyzkoski" width="120vw">
+                </div>
+                <div class="speaker-info">
+                    <h3>Mike Wyzkoski</h3>
+                    <h6>Co Founder Dream Team</h6>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
+                </div>
+
+            </div>
+
+            <div class="speaker2" id="">
+                <!-- <img class="tiles" src="./img/tiles.png" alt="Tiles" width="10vw"> -->
+                <img src="./img/speaker-2.png" alt="Zara Johnson" width="120vw">
+                <div class="speaker-info">
+                    <h3>Zara Johnson</h3>
+                    <h6>CEO WaterBridges</h6>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
```

Comment:
[OPTIONAL] I think it will look nicer and better if the speaker Info is aligned to the left according to what we have in the design template. I mean the `h3`, `h6`, and `p` tags
| Figma Design |Your Design |
| :---:   | :---: | 
| ![image](https://user-images.githubusercontent.com/73854999/203858511-5f26b401-7669-4ae7-bd92-4e26c1bdfcf1.png)|![image](https://user-images.githubusercontent.com/73854999/203858553-bb209110-57d0-4baf-9a78-54484c192b6f.png)|